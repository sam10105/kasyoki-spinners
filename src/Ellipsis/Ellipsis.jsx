import React from 'react';
import './style.css';

const Ellipsis = () => (
  <div className="ellipsis">
    <div />
    <div />
    <div />
  </div>
);

export default Ellipsis;
