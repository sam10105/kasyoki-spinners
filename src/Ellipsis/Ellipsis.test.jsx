import React from 'react';
import { render } from '@testing-library/react';
import Ellipsis from './Ellipsis';

it('should take a snapshot', () => {
  const { asFragment } = render(<Ellipsis />);

  expect(asFragment(<Ellipsis />)).toMatchSnapshot();
});
