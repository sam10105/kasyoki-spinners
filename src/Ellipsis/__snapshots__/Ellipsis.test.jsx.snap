// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="ellipsis"
  >
    <div />
    <div />
    <div />
  </div>
</DocumentFragment>
`;
